zerver/migrations/0565_mark_narrow_to_dm_with_welcome_bot_new_user_as_read.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 5.0.6 on 2024-07-25 13:24
+
+from django.db import connection, migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.utils.timezone import now as timezone_now
+from psycopg2.sql import SQL
+
+
+def mark_narrow_to_dm_with_welcome_bot_new_user_as_read(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    with connection.cursor() as cursor:
+        cursor.execute(
+            SQL(
+                "SELECT MAX(id) FROM zerver_userprofile WHERE is_bot = False AND is_mirror_dummy = False;"
+            )
+        )
+        (max_id,) = cursor.fetchone()
```

Comment:
This should just use the max ID, and we need to run past it in case new users are created while this in running. See `0527_presencesequence.py` for an example.